datalake/src/main/java/com/sequenceiq/datalake/flow/delete/SdxDeleteHandlerSelectors.java

```diff
@@ -0,0 +1,20 @@
+package com.sequenceiq.datalake.flow.delete;
+
+import com.sequenceiq.flow.core.FlowEvent;
+
+public enum SdxDeleteHandlerSelectors implements FlowEvent {
```

Comment:
`SdxDeleteEvent` enum

Reply:
Like for `SdxCreateHandlerSelectors`, I intended this one to gather events used for triggering handlers. `SdxDeleteEvent` - that is also modified in this PR - only contains handler success / failure events that govern the state transitions in the `FlowConfig`.

Reply:
This one has also been removed. :smile: